### PR TITLE
Adding GitHub best practices for project structure

### PR DESCRIPTION
**Pull request by jlcanovas**
This PR addresses the best practices for GitHub project structure, including code of conduct, license and contributing.

The changes proposed by this PR affect the way the project is governed. Besides reviewing the changes, it is recommended to careful deal with the following points:

`CODE_OF_CONDUCT.md`
- [x] Review list of project maintainers and update project's configuration in GitHub as decided

`CONTRIBUTING.md`
- [x] Check indications for issues (either question or proposal) and pull requests
- [x] Check templates for issues at `.github/ISSUE_TEMPLATE`
- [x] Check template for pull requests at `.github/pull_request_tempalte.md`

`GOVERNANCE.md`
- [x] Review list of project maintainers and update project's configuration in GitHub as decided
- [x] Check the period of time to give an answer to an issue
- [x] Check the period of time to give an answer to a pull request
- [x] Check the decision criteria for approving a pull request

`LICENSE.md `
- [x] Check that the indicated license is the one we commented

`CITATION.cff`
- [x] Add missing ORCIDs if needed

**Note that** this PR does not cover the project's configuration at GitHub. Current owners (@jesusc?) of the project should also:
- [x] Update the project description
- [x] Indicate the project's website
- [x] Provide labels
- [x] Decide if the project's home page should include releases, packages and environments

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+We as members, contributors, and leaders pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Need Help?
+If ANY behavior makes you uncomfortable, or you believe it breaches the intent of this code of conduct, please contact a project maintainer:
+
+* [José Antonio Hernández](https://github.com/jlcanovas/) (he/him)
+* [Javier Cánovas](https://github.com/jlcanovas/) (he/him)
+* [Jesús Sánchez](https://github.com/jlcanovas/) (he/him)
```
Comment (jesusc) [inline CODE_OF_CONDUCT.md]: Please, use https://github.com/antolin1/ and https://github.com/jesusc

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+The group of project maintainers commit to give an answer to any issue in a period of time of **48 hours**. 
+
+## Pull Request governance
+
+Both collaborators and project maintainers may propose pull requests. When a collaborator proposes a pull request, is considered contributor.
+
+Pull requests should comply with the template provided. The assignment of labels and assignees to the pull request is the responsibility of the project maintainers.
+
+The group of project maintainers commit to give an answer to any pull request in a period of time of **48 hours**.
```
Comment (jesusc) [inline GOVERNANCE.md]: Let's commit to 1 week.

File: CITATION.cff [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
-}
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Hernández López"
+  given-names: "José Antonio"
+- family-names: "Cánovas Izquierdo"
+  given-names: "Javier Luis"
+  orcid: "https://orcid.org/0000-0002-2326-1700"
+- family-names: "Sánchez Cuadrado"
```
Comment (jesusc) [inline CITATION.cff:10]: Jesús Orcid: https://orcid.org/0000-0001-9755-5616

**Review by jesusc — CHANGES_REQUESTED** [review]
A few simple changes, please.

File: CITATION.cff [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
-  title={ModelSet: a dataset for machine learning in model-driven engineering},
-  author={L{\'o}pez, Jos{\'e} Antonio Hern{\'a}ndez and C{\'a}novas Izquierdo, Javier Luis and Cuadrado, Jes{\'u}s S{\'a}nchez},
-  journal={Software and Systems Modeling},
-  pages={1--20},
-  year={2021},
-  publisher={Springer}
-}
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
```
Comment (jlcanovas) [inline CITATION.cff:3]: @Antolin1 do you want to add your ORCID? If so, what's your ORCID?

File: GOVERNANCE.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Collaborators may propose changes to the project's source code. The mechanism to propose such a change is a GitHub pull request. A collaborator proposing a pull request is considered a contributor. Project maintainers will review and merge pull requests.
+
+## Issue governance
+
+Both collaborators and project maintainers may propose issues. The participation in the issue discussion is open and must follow the [Code of Conduct](CODE_OF_CONDUCT.md).
+
+The group of project maintainers will be responsible for assigning labels to issues, as well as assign the issue to a project maintainer or contributor.
+
+The group of project maintainers commit to give an answer to any issue in a period of time of **48 hours**.
```
Comment (jlcanovas) [inline GOVERNANCE.md:25]: As PR timeframe has been changed, @jesusc do you think we shuold also update the time for issues?

Comment (jesusc) [reply GOVERNANCE.md:25]: I think answering an issue is ok like this.

**Pull request merged**